src/Components/Helper/Error.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
You can remove this component, is not used anymore.

Reply:
Resolved on a56251bb86c0fd869b01be3b7e8da944a136a2a1